#
# This patch remove auto update, suitable for custom fork users
#

 launch_chffrplus.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/launch_chffrplus.sh b/launch_chffrplus.sh
index 41f8a607..69a243e3 100755
--- a/launch_chffrplus.sh
+++ b/launch_chffrplus.sh
@@ -6,11 +6,11 @@ fi
 
 function launch {
   # apply update
-  if [ "$(git rev-parse HEAD)" != "$(git rev-parse @{u})" ]; then
-     git reset --hard @{u} &&
-     git clean -xdf &&
-     exec "${BASH_SOURCE[0]}"
-  fi
+#  if [ "$(git rev-parse HEAD)" != "$(git rev-parse @{u})" ]; then
+#     git reset --hard @{u} &&
+#     git clean -xdf &&
+#     exec "${BASH_SOURCE[0]}"
+#  fi
 
   # no cpu rationing for now
   echo 0-3 > /dev/cpuset/background/cpus
